pkg/releasenotes/releasenotes.go

```diff
@@ -38,15 +41,21 @@ type Client struct {
 
 // NewClient ...
 func NewClient(token string) *Client {
-	client := github.NewClient(nil)
-
+	var client *github.Client
 	// Eventually create an authenticated client
 	if token != "" {
 		ts := oauth2.StaticTokenSource(
 			&oauth2.Token{AccessToken: token},
 		)
 		tc := oauth2.NewClient(context.Background(), ts)
 		client = github.NewClient(tc)
+	} else {
+		// Force use a rate limited client. It will be slow but should overcome rate limiting issues.
+		rateLimiter, err := github_ratelimit.NewRateLimitWaiterClient(nil)
```

Comment:
Force a rate limited github client when no token is passed.

Reply:
Nice